### DISCUSSION TITLE
Pitch Adjust [Feature Suggestion]

### DISCUSSION THREAD

**Discussion by darkzip** [PaulWoitaschek/Voice#1792] [Ideas]
Adding a voice pitch adjustment would be a decent feature for the occasional grating reader.

**Comment by riparise** [comment]
Second this. First of all, thanks for the amazing app.

It would be nice to have the possibility to change not only the playback speed, but the pitch as well (for example in Musicolet / Newpipe).

I do find some narrator's voice a bit unpleasant, and being able to change the pitch normally is a nice option.

It would be cool to be able to change it globally or for a specific book, in a way that it remembers the playback speed and pitch settings next time you play the book.

That all, thanks for you attention
